universal-login-sdk/lib/api/FutureWalletFactory.ts

```diff
@@ -56,15 +55,7 @@ export class FutureWalletFactory extends MineableFactory {
   }
 
   createDeployingWallet(serializedDeployingWallet: SerializedDeployingWallet): DeployingWallet {
-    const {deploymentHash, contractAddress, name, privateKey} = serializedDeployingWallet;
-    return {
-      ...serializedDeployingWallet,
-      waitForTransactionHash: this.createWaitForTransactionHash(deploymentHash),
-      waitToBeSuccess: async () => {
-        await this.createWaitToBeSuccess(deploymentHash)();
-        return new DeployedWallet(contractAddress, name, privateKey, this.sdk);
-      },
-    };
+    return new DeployingWallet(serializedDeployingWallet, this.sdk);
```

Comment:
Since we extracted DeployingWallet, FutureWalletFactory propbably shouldn't extends MineableFactory'

Reply:
#nextPR

Reply:
#nextPR Inline createDeployingWallet

Reply:
Done here

Reply:
done here